met-web/src/components/engagement/view/SurveyBlock.tsx

```diff
@@ -2,12 +2,11 @@ import React, { useContext } from 'react';
 import { Button, Grid, Typography } from '@mui/material';
 import { MetPaper } from '../../common';
 import './EngagementContent.scss';
-import { useNavigate } from 'react-router-dom';
 import { ActionContext } from './ActionContext';
+import { SurveyBlockProps } from './types';
 
-const SurveyBlock = () => {
+const SurveyBlock = (props: SurveyBlockProps) => {
```

Comment:
please always destruct the props so the reader would see them clearly. In this case instead of props, { openModal }